src/server/src/main/docker/configure-persistence.sh

```diff
@@ -3,20 +3,29 @@
 case ${REAPER_STORAGE_TYPE} in
     "cassandra")
 cat <<EOT >> /etc/cassandra-reaper.yml
+activateQueryLogger: ${REAPER_ACTIVATE_QUERY_LOGGER}
+
 cassandra:
   clusterName: ${REAPER_CASS_CLUSTER_NAME}
   contactPoints: ${REAPER_CASS_CONTACT_POINTS}
   keyspace: ${REAPER_CASS_KEYSPACE}
 EOT
 
-if [ "true" = "${REAPER_CASS_AUTH_ENABLED}" ]; then
+    if [ "true" = "${REAPER_CASS_AUTH_ENABLED}" ]; then
```

Comment:
The new file looks a bit off with the indentations. Perhaps a series of comment lines like `# BEGIN... # END`?

Reply:
I was on the fence with the indentations as it was. I have added in the comments as suggested.